comp/trace/config/otlp_serverless.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build serverless
+
+package config
+
+import (
+	corecompcfg "github.com/DataDog/datadog-agent/comp/core/config"
+	coreconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/trace/config"
+)
+
+func setOtlpReceiver(c *config.AgentConfig, core corecompcfg.Component, grpcPort int) error {
+	c.OTLPReceiver = &config.OTLP{
+		BindHost:               c.ReceiverHost,
+		GRPCPort:               grpcPort,
+		MaxRequestBytes:        c.MaxRequestBytes,
+		SpanNameRemappings:     coreconfig.Datadog.GetStringMapString("otlp_config.traces.span_name_remappings"),
+		SpanNameAsResourceName: core.GetBool("otlp_config.traces.span_name_as_resource_name"),
+		ProbabilisticSampling:  core.GetFloat64("otlp_config.traces.probabilistic_sampler.sampling_percentage"),
```

Comment:
Why are we not setting attributesTranslator ?

Reply:
I believe I was wrong on doing this change on this PR. This PR focused on removing prometheus from the serverless binary. In this case these changes are more related to otlp which should be handle on a separate PR

Reply:
Done https://github.com/DataDog/datadog-agent/pull/24854/commits/e6cf9f78c89920af3125cbcf051011dd659f4739